### ISSUE TITLE
Backend SQLite3 change directory ownership

### ISSUE THREAD

**Issue by alebeta90** [PowerDNS/pdns-ansible#71]
Hi friends,

Recently I had been deploying my slaves dns servers with version PowerDNS 4.2 and it was having troubles to write changes in the database. After asking for help in the IRC group, I got the follow answer

> 4.2 changed the default SQLite journal mode to WAL, so the process has to be able to create db.sqlite-shm and db.sqlite-wal files.

and the suggestion was to change the ownership of powerdns db directory to user `pdns` and group `pdns`. I did it in my Ansible PB as adding this:

``` yaml
        - name: Change /var/lib/powerdns/ ownership to pdns user and group
          file:
            path: /var/lib/powerdns/
            owner: pdns
            group: pdns
``` 

I think it would be great if this piece of code  could be integrated in the role. I could help to do the PR.

Thanks and all the best

**Comment by atosatto** [comment]
Hi @alebeta90, thanks for your detail report.
Indeed the role is not correctly handling the permissions for the new 4.2.x PowerDNS release.

> I could help to do the PR.

Please go ahead submitting a PR. I will take care or review and merge it ASAP.

**Comment by alebeta90** [comment]
Hi @atosatto 

could you have a look at my change in the PB. For some reason in the CI it is giving me an error related the lint, I run some lint test but can not see the error.

Thanks

**Comment by pieterlexis** [comment]
Should be fixed with #82
